src/main/java/io/excaliburfrc/robot/subsystems/Climber.java

```diff
@@ -39,32 +85,86 @@ public void close() {
     }
   }
 
-  public void activateLeader(MotorMode m) {
-    motor.set(m.dutyCycle);
+  public void activateMotors(MotorMode m) {
+    leftMotor.set(m.dutyCycle);
+    rightMotor.set(m.dutyCycle);
   }
 
   public void openAngler() {
-    anglerPiston.set(DoubleSolenoid.Value.kForward);
+    piston.set(DoubleSolenoid.Value.kForward);
   }
 
   public void closeAngler() {
-    anglerPiston.set(DoubleSolenoid.Value.kReverse);
+    piston.set(DoubleSolenoid.Value.kReverse);
+  }
+
+  private Command reachSideHeight(
+      SparkMaxPIDController controller,
+      RelativeEncoder encoder,
+      double height,
+      double mg,
+      double angle) {
+    trapezoidGoal.position = height;
+    trapezoidInit.position = encoder.getPosition();
```

Comment:
I don't think this is the correct use, and not what I meant with caching the profiles (the `TrapezoidProfile` objects).